sky/execution.py

```diff
@@ -287,11 +291,19 @@ def launch(
     optimize_target: OptimizeTarget = OptimizeTarget.COST,
     detach_run: bool = False,
     no_setup: bool = False,
-):
-    """Launch a sky.DAG or sky.Task (rerun setup if cluster exists).
+) -> None:
+    # NOTE(dev): Keep the docstring consistent between the Python API and CLI.
+    """Launch a task.
+
+    The task's setup and run commands are executed under the task's workdir
+    (when specified, it is synced to remote cluster).  The task undergoes job
+    queue scheduling on the cluster.
+
+    Currently, the first argument must be a single-node DAG (i.e., a single
```

Comment:
nit: The first argument accepts a sky.Task, or (more advanced usage) a sky.Dag; in the latter case, currently it must be ...

Reply:
Good catch! Fixed. Thanks!

Reply:
Good catch! Fixed. Thanks!